validator/coordinator/src/test/java/tech/pegasys/artemis/validator/coordinator/Eth1DataManagerTest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.validator.coordinator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.primitives.UnsignedLong;
+import java.util.List;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.artemis.datastructures.blocks.Eth1Data;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.pow.event.CacheEth1BlockEvent;
+import tech.pegasys.artemis.storage.events.SlotEvent;
+import tech.pegasys.artemis.util.SSZTypes.SSZList;
+import tech.pegasys.artemis.util.config.Constants;
+
+public class Eth1DataManagerTest {
+
+  private final EventBus eventBus = new EventBus();
+  private final UnsignedLong genesisTime = UnsignedLong.ZERO;
+
+  static {
+    Constants.SECONDS_PER_ETH1_BLOCK = UnsignedLong.valueOf(10);
+    Constants.ETH1_FOLLOW_DISTANCE = UnsignedLong.valueOf(100);
+    Constants.SLOTS_PER_ETH1_VOTING_PERIOD = 10;
+    Constants.SECONDS_PER_SLOT = 10;
```

Comment:
Might be a good idea to set distinct values for the various constants to guard against bugs where we use the wrong constant.

Reply:
done.